recipe/meta.yaml

```
@@ -12,11 +12,12 @@ source:
   patches:
     - 0001-Use-CMAKE_SYSTEM_PROCESSOR-instead-of-CMAKE_LIBRARY_.patch
     - 0001-Add-missing-unistd-include.patch
+    - 0001-cut-outdated-apple-version-checks.patch
```

nit: document the patch provenance in `meta.yaml`
```suggestion
    # backport https://github.com/facebook/folly/commit/e3dc03f5621935bf9caabadb9f46712f561747e9
    - 0001-cut-outdated-apple-version-checks.patch
```